### ISSUE TITLE
TypeError: n.replaceAll is not a function

### ISSUE THREAD

**Issue by thejhh** [hangovergames/create-backend#5]
```
TypeError: n.replaceAll is not a function
    at /Users/jhh/git/heusalagroup/create-backend/dist/create-backend.js:456:100
    at Array.forEach (<anonymous>)
    at Function.copyTextFileWithReplacements (/Users/jhh/git/heusalagroup/create-backend/dist/create-backend.js:456:73)
    at Function.copyTextFileWithReplacementsIfMissing (/Users/jhh/git/heusalagroup/create-backend/dist/create-backend.js:458:39)
    at /Users/jhh/git/heusalagroup/create-backend/dist/create-backend.js:917:64
    at Array.forEach (<anonymous>)
    at /Users/jhh/git/heusalagroup/create-backend/dist/create-backend.js:917:24
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

```
$ node --version
v14.20.1

```

**Comment by thejhh** [comment]
This is because NodeJS v14 does not have `"".replaceAll()`.

**Comment by thejhh** [comment]
Fixed problems with HG's `replaceAl()` so that can be used again now as a replacement:

https://github.com/heusalagroup/fi.hg.core/issues/29

**Comment by thejhh** [comment]
Fixed in version 1.0.6
